### ISSUE TITLE
Upgrade bundled PhoneGap from 0.9.5

### ISSUE THREAD

**Issue by GoogleCodeExporter** [erhanselman/mobile-web-development-with-phonegap#34]
```
Currently 0.9.5 is included. 0.9.6 is the current rev


```

Original issue reported on code.google.com by `paul.beu...@gmail.com` on 19 Jul 2011 at 6:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 19 Jul 2011 at 6:46
- Changed title: **Upgrade bundled PhoneGap from 0.9.5**

**Comment by GoogleCodeExporter** [comment]
Original comment by `paul.beu...@gmail.com` on 19 Jul 2011 at 6:59
- Added labels: **Milestone-Release1.2.4**

**Comment by GoogleCodeExporter** [comment]
```
Fixed with 1.2.4
```

Original comment by `paul.beu...@gmail.com` on 19 Aug 2011 at 4:30
- Changed state: **Fixed**
